### DISCUSSION TITLE
google cloud, compute engine forwards to disk

### DISCUSSION THREAD

**Discussion by choonho** [cloudforet-io/community#101] [Bug Report] [answered]
![image](https://user-images.githubusercontent.com/3993088/212826355-58d56622-cd24-43bb-bc9c-e80af869f5ae.png)

If I click "ComputeEngine", this link forwards to Disk not Instance

**Comment by whdalsrnt** [comment, marked as answer]
Hi, @choonho 
The above issue is not a bug, but an intended design.

Cloud services shown on the `Asset Inventory > Cloud Service` page have the following policies.
1. One Cloud Service Group shows up to two Primary Types.
2. Up to two types of information are displayed at the bottom of the `Cloud Service Card`, and you can move to the detailed page through the link.
3. When you click the main 'Cloud Service Card', you will be directed to the detail page of the cloud service with the fastest sorted name.

This issue happened because `Disk` is faster than `Instance` in alphabetical order as a result of policy 3.

I agree with what you asked, so we will think about it one more time.

Thank you.
